### PR TITLE
POWERMON-199: fix gather power-monitoring-operator info

### PR DESCRIPTION
**Pull request by vimalk78**
fix gather power-monitoring-operator info

### CODE REVIEW

File: must-gather/gather-power-monitoring-operator-info [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+source "${SCRIPT_DIR}/common"
+
+BASE_COLLECTION_PATH=$1
+
+KEPLER_OPERATOR_NS="openshift-operators"
+
+get_catalogsource() {
+	log "getting catalogsource info for kepler-operator"
+	run oc -n "$KEPLER_OPERATOR_NS" get catalogsource kepler-operator-catalog -oyaml "$POWER_MONITORING_OPERATOR_INFO_DIR/kepler-operator-catalogsource.yaml"
```
Comment (vprashar2929) [inline must-gather/gather-power-monitoring-operator-info:29]: Shouldn't we remove this as the operator would be from Red Hat certified Operators catalog source right?

Comment (vimalk78) [reply must-gather/gather-power-monitoring-operator-info:29]: kept it for community operator

Comment (vprashar2929) [reply must-gather/gather-power-monitoring-operator-info:29]: but `gather-power-monitoring-operator-info` will never be called up for community operator

**Review by vprashar2929 — APPROVED** [review]

**Comment by vprashar2929** [comment]
It would be nice if we could address the previous issue with must-gather: https://github.com/sustainable-computing-io/kepler-operator/pull/261#discussion_r1346664106

File: must-gather/gather-olm-info [hunk 2]
```diff
@@ -22,17 +22,25 @@ source "${SCRIPT_DIR}/common"
 
 BASE_COLLECTION_PATH=$1
 
-KEPLER_OPERATOR_NS="openshift-operators"
 KEPLER_OPERATOR_LABELS="operators.coreos.com/kepler-operator.openshift-operators"
+POWER_MONITORING_OPERATOR_LABELS="operators.coreos.com/power-monitoring-operator.openshift-operators"
```
Comment (sthaha) [inline must-gather/gather-olm-info:26]: For the record I am okay with this but we need to address operator getting installed to `ns` other than `openshift-operators` (esp on 4.14+) in a different PR. Perhaps worth a bug @vprashar2929 ?

Comment (vimalk78) [reply must-gather/gather-olm-info:26]: if that is really needed, lets do it in another PR

Comment (sthaha) [reply must-gather/gather-olm-info:26]: yes, as already mentioned :-)

Comment (vimalk78) [reply must-gather/gather-olm-info:26]: ok, can we merge this?

**Review by sthaha — APPROVED** [review]

**Pull request merged**